rocketpy/motors/Tank.py

```diff
@@ -7,15 +7,40 @@
 from abc import ABC, abstractmethod
 
 from rocketpy.Function import Function, funcify_method
-from rocketpy.utilities import except_negative
 
 
 class Tank(ABC):
-    def __init__(self, name, geometry, gas, liquid=0):
+    """Abstract Tank class that defines a tank object for a rocket motor, so
+    that it evaluates useful properties of the tank and its fluids, such as
+    mass, volume, fluid flow rate, center of mass, etc.
+    """
+
+    def __init__(self, name, geometry, flux_time, gas, liquid, discretize=100):
+        """Initialize Tank class.
+
+        Parameters
+        ----------
+        name : str
+            Name of the tank.
+        geometry : TankGeometry
+            Geometry of the tank.
+        flux_time : tuple
+            Tuple containing start and final times of the tank flux.
+        gas : Fluid
```

Comment:
Shouldn't this work like `burn_time` from [#337](https://github.com/RocketPy-Team/RocketPy/pull/337)?

Reply:
Solved by [4f0835a](https://github.com/RocketPy-Team/RocketPy/pull/354/commits/4f0835ac7ac1f97d1e9f45f3001b11426ea202b0).